org.opencpn.OpenCPN.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2018 Alec Leamas
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+
+# This manifest is used to build the official flathub package. 
+
+
+app-id: org.opencpn.OpenCPN
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
+sdk: org.freedesktop.Sdk
+command: opencpn.sh
+
+rename-desktop-file: opencpn.desktop
+rename-icon: opencpn
+rename-appdata-file: opencpn.appdata.xml
+
+finish-args:
+    - --socket=x11
+    - --socket=pulseaudio
+    - --filesystem=home
+    - --share=network
+    - --device=all
+
+add-extensions:
+    org.opencpn.OpenCPN.Plugin:
+        directory: extensions
+        merge-dirs: lib/opencpn;share/opencpn/plugins;share/locale
+        subdirectories: true
+        no-autodownload: true
+        autodelete: false
+
+modules:
+    - name: mesa-libGLU
```

Comment:
It's also available in shared-modules: https://github.com/flathub/shared-modules/

Reply:
Tried using it, but the wxWidgets build fails, it doesn't see the glu.h header. Guess that the shared-modules are not guaranteed to be installed at that point (?)

Reply:
It works for me locally; pushed a change to your branch. Let's see what CI tells.